electron/InMemoryTransport.js

```diff
@@ -11,7 +11,9 @@ module.exports = class InMemoryTransport extends Transport {
     const MESSAGE_SYMBOL = Symbol.for('message');
     const message = info[MESSAGE_SYMBOL];
     const strippedMessage = stripAnsi(message);
-    this.loggedMessages.push(strippedMessage);
+    console.log('info: ', info);
+    console.log('level: ', stripAnsi(info.level));
```

Comment:
I'm guessing you were using these `console.log`s to develop/debug, but we should remove them now.